Xamarin.Forms.Platform.Android/AppCompat/FrameRenderer.cs

```diff
@@ -286,5 +290,22 @@ void UpdateShadow()
 			else
 				CardElevation = 0f;
 		}
+
+		void UpdateCornerRadius()
+		{
+			if (_defaultCornerRadius == -1f)
+			{
+				_defaultCornerRadius = Radius;
+			}
+
+			float cornerRadius = Element.CornerRadius;
+
+			if (cornerRadius == -1f)
```

Comment:
I'd write `if (cornerRadius < 0)`, so you don't have to think about what happens if the user sets a value of, say, `-2`